src/compiler/checker.ts

```diff
@@ -28290,6 +28290,25 @@ namespace ts {
             return true;
         }
 
+        function getEnclosingClassFromThisParameter(node: Node): InterfaceType | undefined {
+            let thisType = getThisTypeFromNodeContext(node);
+

```

Comment:
extremely minor style preference: I think the blank lines in this function should be deleted.